src/driver/controller.go

```diff
@@ -169,6 +169,8 @@ func (s *ControllerService) ControllerPublishVolume(ctx context.Context, req *pr
 			code = codes.FailedPrecondition
 		case volumes.ErrAttachLimitReached:
 			code = codes.ResourceExhausted
+		case volumes.ErrLockedServer:
```

Comment:
I think this part can be included in a PR about fixing the locked server errors CSI-driver receives from the hcloud.